operate/webapp/src/main/java/io/camunda/operate/webapp/backup/BackupConfig.java

```diff
@@ -24,4 +27,35 @@ public ThreadPoolTaskExecutor getTaskExecutor() {
     executor.initialize();
     return executor;
   }
+
+  @Bean
+  public static BackupRepositoryProps backupRepositoryProps(
+      final OperateProperties operateProperties) {
+    return props(operateProperties.getVersion(), operateProperties.getBackup());
+  }
+
+  public static BackupRepositoryProps props(
+      final String operateVersion, final BackupProperties operateProperties) {
+    return new BackupRepositoryProps() {
+      @Override
+      public String version() {
+        return operateVersion;
+      }
+
+      @Override
+      public String repositoryName() {
+        return operateProperties.getRepositoryName();
+      }
+
+      @Override
+      public int snapshotTimeout() {
+        return operateProperties.getSnapshotTimeout();
+      }
+
+      @Override
+      public Long incompleteCheckTimeoutInSeconds() {
+        return operateProperties.getIncompleteCheckTimeoutInSeconds();
+      }
+    };
```

Comment:
:question: Why an anonymous class? There's a `BackupRepositoryPropsImpl` class, no?

Reply:
Unfortunately `BackupRepositoryPropsImpl` take a reference of the properties when the class is instantiated, which means that when testing the mocks are setup after the class was already instantiated. 
This implementation delegates to the properties, so it work even mocking the properties afterwards.
I will probably remove the record if it's not in use anymore in the next PRs